README.md

```diff
@@ -129,10 +129,6 @@ end)
 
 ```lua
 RegisterNetEvent('qb-multicharacter:client:spawnLastLocation', function(coords, cData)
-    local result = lib.callback.await('ps-housing:cb:GetOwnedApartment', source, cData.citizenid)
```

Comment:
You could leave this, how come you removed it?

Reply:
Well, if you don't ensure `ox_lib` before `qb-multicharacter`, it cannot run the callback (at least on my part) 😟
That's also why I originally added the dependencies part...